### PR TITLE
Handle v3 pagination

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: tests/python/integration/pytest.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[pytest]
+usefixtures = plugin dss_target
```
Comment (JaneBellaiche) [inline tests/python/integration/pytest.ini:2]: what is this for ?

File: python-lib/odata_constants.py [hunk 2]
```diff
@@ -4,21 +4,24 @@ class ODataConstants(object):
...
     DATA_RESULTS = "results"
     ENTITYSETS = "EntitySets"
+    FORMAT = "$format"
     INSTANCE = "odata_instance"
+    JSON = "json"
     LIST_TITLE = "odata_list_title"
     LOGIN = "sap-odata_login"
-    NEXT_LINK = "@odata.nextLink"
+    NEXT_LINK_V3 = "odata.nextLink"
+    NEXT_LINK_V4 = "@odata.nextLink"
```
Comment (JaneBellaiche) [inline python-lib/odata_constants.py:13]: what's the difference between those?

Comment (alexbourret) [reply python-lib/odata_constants.py:13]: This is one example of the tiny differences between the v3 and v4 odata protocol. V3 sends you the last segment of the next page's URL using the "odata.nextLink"
v3

while v4 sends you the full URL using "@odata.nextLink" key...
v4

File: tests/python/integration/pytest.ini (hunk 1 above)
Comment (alexbourret) [reply tests/python/integration/pytest.ini:2]: This is required to run the integrations tests. It is copied from the plugin repo template.

**Review by JaneBellaiche — APPROVED** [review]
LGTM 👍

**Pull request merged**